fastly-compute/geoip/.github/workflows/deploy-geoip.yaml

```diff
@@ -0,0 +1,18 @@
+name: Deploy Compute@Edge GeoIp
+on:
+  push:
+    branches: [main]
```

Comment:
while testing you can make this `branches: "*"` and set back to main when satisfied.

Reply:
this gave me a syntax error, but I found by [removing the branches value](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#on:~:text=Using%20a%20single,the%20workflow%27s%20repository) the workflow will run when a push is made to any branch.